garden/ros_gz_project_template_guide.md

```diff
@@ -0,0 +1,97 @@
+# Guide to `ros_gz_project_template` for ROS 2 and Gazebo Development
+
+In this guide, you will learn how to use the `ros_gz_project_template` to create a (recommended) structured workspace or improve your existing workspace for your ROS 2 and Gazebo projects. This template offers a consistent layout, automated build process, and integration with both ROS 2 and Gazebo, enabling you to focus on developing your robotics applications.
+
+## Installation Steps
+
+1. If you are starting a project, create new workspace or go to your existing project's source directory:
+
+```bash
+    mkdir -p ~/project_ws/src
+    cd ~/project_ws/src
+```
+
+1. Directly `Use this template` and create your project repository on Github.
+
+   ![use_template](tutorials/ros2_integration/use_template.png)
+
+   Or start by cloning the `ros_gz_project_template` repository:
```

Comment:
I moved the name to the first mention above so it's clear these are talking about the same repository
```suggestion
   Or start by cloning the template repository:
```